python/core/auto_generated/pointcloud/qgspointcloudrenderer.sip.in

```diff
@@ -156,6 +156,24 @@ Returns the feedback object used to cancel rendering
 .. versionadded:: 3.20
 %End
 
+    void setElevationMap( QgsElevationMap *elevationMap /Transfer/ );
```

Comment:
How do you feel about hiding these methods from sip? There's a good chance we may want to shuffle them around in future (eg moving them to QgsRenderContext), and I wouldn't want to be constrained by stable API issues if/when we do this.